lib/cli.zsh

```diff
@@ -281,8 +281,8 @@ multi == 1 && length(\$0) > 0 {
 
   local zdot="${ZDOTDIR:-$HOME}"
   awk "$awk_script" "$zdot/.zshrc" > "$zdot/.zshrc.new" \
-  && command mv -f "$zdot/.zshrc" "$zdot/.zshrc.bck" \
-  && command mv -f "$zdot/.zshrc.new" "$zdot/.zshrc"
```

Comment:
Hi Ben, you were actually very close: you don't need to use glob modifiers, since these are also applicable to parameters and they don't use a subcommand:
```suggestion
  local zdot="${ZDOTDIR:-$HOME}"
  local zshrc="${${:-"${zdot}/.zshrc"}:A}"
  awk "$awk_script" "$zshrc" > "${zdot}/.zshrc.new" \
  && command cp -f "$zshrc" "${zdot}/.zshrc.bck" \
  && command mv -f "${zdot}/.zshrc.new" "$zshrc"
```
which is less ugly-looking by introducing a new `local` variable.

The `mv -f` was intentional here, but I left the `cp`. Since the `.bck` file is not getting deleted though, it should be forced to override the previous ones if existing.